cypress/support/fragments/inventory/inventoryInstance.js

```diff
@@ -108,6 +108,7 @@ const closeDetailsView = Button({ icon: 'times' });
 const quickMarcEditorPane = Section({ id: 'quick-marc-editor-pane' });
 const filterPane = Section({ id: 'pane-filter' });
 const inputSearchField = TextField({ id: 'input-inventory-search' });
+const authSearchResultsPane = Section({ id: 'authority-search-results-pane' });
```

Comment:
'authSearchResultsPane' is assigned a value but never used. Allowed unused vars must match /React/u